artemis-server/src/test/resources/ConfigurationTest-xinclude-config-address-settings.xml

```diff
@@ -14,7 +14,10 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<address-settings xmlns="urn:activemq:core">
+<address-settings xmlns="urn:activemq:core"
```

Comment:
should not need to have any changes to existing test cases, any such changes suggest break changes

Reply:
As shown in the broker-connections and security-settings test files, the additional namespace attributes are not necessary and do not represent a breaking change.

Reply:
then there should not have been a need for any change to these files.

Reply:
existing test cases and configs should remain untouched.

Reply:
As I stated, these changes are not needed for the tests to pass. These changes simply add other XML include paths to the existing test suite.

Reply:
So don't change them.

Reply:
I'm currently working on a commit that does just that.

Reply:
Reverted all changes to the prior ConfigurationFile-xinclude-....xml unit test resource files. Added a separate unit test class and many modular include files for all top-level, non-trivial element types while still passing the existing FileConfigurationTest and ConfigurationImplTest scenarios.

The ConfigurationImplTest#testSerialize() method makes a questionable assumption regarding broker plugins in test configurations but I worked around it to preserve the existing unit test while also including a modular broker plugins file.